pkgs/by-name/di/digikam/package.nix

```diff
@@ -110,6 +111,13 @@ stdenv.mkDerivation rec {
     cuda_cudart
   ]);
 
+  postPatch = ''
+    substituteInPlace \
+      core/dplugins/bqm/custom/userscript/userscript.cpp \
+      core/utilities/import/backend/cameracontroller.cpp \
+      --replace-fail \"/bin/bash\" \"${lib.getExe bash}\"
```

Comment:
```suggestion
      --replace-fail "/bin/bash" "${lib.getExe bash}"
```

Reply:
This was quite deliberate, so that it really only matches the string `"/bin/bash"` (including quotes) in the source and doesn't introduce false positives later.